services/graphite/parser.go

```diff
@@ -225,6 +227,18 @@ func (t *template) Apply(line string) (string, map[string]string, string, error)
 		tags[k] = append(tags[k], v)
 	}
 
+	// See if an invalid combination has been specified in the template:
+	for _, tag := range t.tags {
+		if tag == "measurement*" {
+			wildcard_measurement_specified = true
+		} else if tag == "field*" {
+			wildcard_field_specified = true
+		}
+	}
+	if wildcard_field_specified && wildcard_measurement_specified {
```

Comment:
Go style is to use `mixedCaps` instead of underscores: https://golang.org/doc/effective_go.html#mixed-caps

Can you rename to something like `hasFieldWildcard` and `hasMeasurementWildcard`?